workflows/autometa_gtdb_flagged.sh

```diff
@@ -0,0 +1,538 @@
+#!/usr/bin/env bash
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=16
+#SBATCH --partition=queue
+#SBATCH --time=14-00:00:00
+#SBATCH --error=autometa.%J.err
+#SBATCH --output=autometa.%J.out
+
+# Exit if a process returns a non-zero exit code
+set -e
+
+usage()
+{
+   # Display Help
+   echo "-h     Display help menu"
+   echo
+   echo "Required flags:"
+   echo "-o     Directory path to where the output will go"
+   echo "-a     File path to your assembly file (include file name)"
+   echo "-s     Simple name to prefix output files"
+   echo "-n     Path to NCBI databases directory"
+   echo "-g     Path to GTDB databases directory"
```

Comment:
The `autometa_flagged.sh` workflow should also be changed to comply with the `--dbdir` and `--dbtype` changes.